pkg/kubelet/cm/dra/plugin/plugins_store.go

```diff
@@ -17,18 +17,59 @@ limitations under the License.
 package plugin
 
 import (
+	"context"
+	"net"
 	"sync"
 
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/connectivity"
+	"google.golang.org/grpc/credentials/insecure"
 	utilversion "k8s.io/apimachinery/pkg/util/version"
+	"k8s.io/klog/v2"
 )
 
 // Plugin is a description of a DRA Plugin, defined by an endpoint
 // and the highest DRA version supported.
 type Plugin struct {
+	sync.RWMutex
+	conn                    *grpc.ClientConn
 	endpoint                string
 	highestSupportedVersion *utilversion.Version
 }
 
+func (p *Plugin) CreateGRPCConn() error {
+	network := "unix"
+	klog.V(4).InfoS(log("creating new gRPC connection"), "protocol", network, "endpoint", p.endpoint)
+
+	p.Lock()
+	defer p.Unlock()
```

Comment:
If we do what I said previously, the top of this call can be replaced with:
```
	p.Lock()
	defer p.Unlock()
	
	if p.conn != nil {
		return p.conn, nil
	}

	network := "unix"
	klog.V(4).InfoS(log("creating new gRPC connection"), "protocol", network, "endpoint", p.endpoint)
	...
```

Reply:
done in ef1a07b143e